hypothesis-python/docs/reference/reproducing-failures.rst

```diff
@@ -1,19 +1,8 @@
-====================
 Reproducing failures
 ====================
```

Comment:
My heading ordering is single =, single -, single ~ (and if you need to go beyond that you're probably doomed anyway). I try to avoid double headers for anything, but I think I might be the odd one out here, because e.g. the python rst guide recommends double headers for chapters. In defense of single headers, the python docs have *way* more structure than we do!